app/services/balance_transactions/creator.rb

```diff
@@ -0,0 +1,25 @@
+class BalanceTransactions::Creator
+  Params = Struct.new(:date, :comment, :amount, :user_id)
+
+  def initialize(params)
+    @params = Params.new(params[:date], params[:comment], params[:amount], params[:user_id])
```

Comment:
@marderer what do you think about going with list of instance variables instead of keeping everything in `@params`?

It's easier to deal with `@date`, `@comment` than with `@params[:date]` and `@params[:comment]`

Reply:
So, we can have something like that:
```ruby
class BalanceTransactions::Creator
  def initialize(date:, comment:, amount:, user_id:)
    @date = date
    @comment = comment
    @amount = amount
    @user_id = user_id
  end
```
+ attr_reader can be used